src/compiler/typescript_to_rust/generators/azle_into_js_value_trait.ts

```diff
@@ -210,6 +210,12 @@ export function generateAzleIntoJsValueTrait(): Rust {
             }
         }
 
+        impl AzleIntoJsValue for usize {
+            fn azle_into_js_value(self, context: &mut boa_engine::Context) -> boa_engine::JsValue {
+                boa_engine::JsValue::BigInt(self.into())
```

Comment:
I'm not sure this should be a BigInt, let's discuss

Reply:
I've changed this to nat32.